examples/setup_run.py

```diff
@@ -176,19 +201,28 @@ def setup_trainer_and_run(cfg: DictConfig, setup_plotter: SetupPlotterFn,
                          n_intermediate_distributions=cfg.fab.n_intermediate_distributions,
                          transition_operator=transition_operator,
                          loss_type=cfg.fab.loss_type)
-    optimizer = torch.optim.AdamW(flow.parameters(), lr=cfg.training.lr)
+    optimizer = torch.optim.Adam(flow.parameters(), lr=cfg.training.lr)
     # scheduler = torch.optim.lr_scheduler.ExponentialLR(optimizer, gamma=0.995)
     scheduler = None
 
-
     # Create buffer if needed
     if cfg.training.use_buffer is True:
         buffer = setup_buffer(cfg, fab_model)
     else:
         buffer = None
+    if chkpt_dir is not None:
+        map_location = "cuda" if torch.cuda.is_available() and cfg.training.use_gpu else "cpu"
+        fab_model.load(os.path.join(chkpt_dir, "model.pt"), map_location)
+        opt_state = torch.load(os.path.join(chkpt_dir, 'optimizer.pt'), map_location)
+        optimizer.load_state_dict(opt_state)
+        if buffer is not None:
+            buffer.load(path=os.path.join(chkpt_dir, 'buffer.pt'))
+        print(f"\n\n****************loaded checkpoint: {chkpt_dir}*******************\n\n")
```

Comment:
This should print if you expect a checkpoint to be loaded (i.e. if I run `make mw_fab_buffer` once and then run it again, on the second time it loads the checkpoint from the first time).